tmail-backend/tmail-third-party/openpaas/src/test/resources/docker-openpaas-setup.yml

```diff
@@ -0,0 +1,97 @@
+version: "2"
+
+networks:
+  emaily:
+    driver: bridge
+
+services:
+  openpaas:
+    hostname: esn
+    image: linagora/esn:branch-1.11
+    ports:
+      - "8080:8080"
+    environment:
+      - SABRE_ENV=dev
+      - PROVISION=true
+      - MONGO_HOST=mongo
+      - MONGO_PORT=27017
+      - MONGO_DBNAME=esn_docker
+      - MONGO_TIMEOUT=100000
+      - REDIS_HOST=redis
+      - REDIS_PORT=6379
+      - AMQP_HOST=rabbitmq
+      - AMQP_PORT=5672
+      - AMQP_PROTOCOL=amqp
+      - AMQP_USERNAME=guest
+      - AMQP_PASSWORD=guest
+      - ELASTICSEARCH_HOST=elasticsearch
+      - ELASTICSEARCH_PORT=9200
+      - WEB_PORT=8080
+      - DAV_SERVER_HOST=esn_sabre
+      - DAV_SERVER_PORT=80
+    depends_on:
+      - mongo
+      - redis
+      - rabbitmq
+      - elasticsearch
+      - sabre_dav
+    networks:
+      - emaily
+
+  redis:
+    image: redis:latest
+    ports:
+      - "6379:6379"
+    networks:
+      - emaily
+
+  rabbitmq:
+    image: rabbitmq:3.13.3-management
+    hostname: esn-rabbit
+    ports:
+      - 5672:5672
+      - 15672:15672
+    networks:
+      - emaily
+
+  mongo:
+    image: mongo
+    ports:
+      - "27017:27017"
+    networks:
+      - emaily
+
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:6.3.2
+    ports:
+      - "9200:9200"
+    environment:
+      - discovery.type=single-node
+    networks:
+      - emaily
+
+  sabre_dav:
+    hostname: esn_sabre
+    image: linagora/sabre:lng-12-02-2025
+    ports:
+      - "8001:80"
+    environment:
+      - SABRE_ENV=dev
+      - SABRE_MONGO_HOST=mongo
+      - SABRE_MONGO_PORT=27017
+      - ESN_MONGO_HOST=mongo
+      - ESN_MONGO_PORT=27017
+      - ESN_MONGO_DBNAME=esn_docker
+      - MONGO_TIMEOUT=100000
+      - ESN_HOST=esn
+      - ESN_PORT=8080
+      - REDIS_HOST=redis
+      - REDIS_PORT=6379
+      - AMQP_HOST=rabbitmq
+      - AMQP_PORT=5672
+      - SABRE_ADMIN_LOGIN=admin
+      - SABRE_ADMIN_PASSWORD=secret123
+      - OPENPASS_BASIC_AUTH=YWRtaW5Ab3Blbi1wYWFzLm9yZzpzZWNyZXQ=
+    networks:
+      - emaily
+      -
```

Comment:
We need more healthcheck if not we will get more unstable tests in future

Reply:
Agree, but I want to keep PRs as small as possible preferably below 500 lines of code.

I created a ticket here: https://github.com/linagora/tmail-backend/issues/1537